joins/challenges/default.json

```diff
@@ -69,361 +69,57 @@
         },
 
 
+{% for i in range(idx_suffix|length) %}
         {
-          "operation": "esql_lookup_join_1k_keys_limit1",
+          "operation": "esql_lookup_join_{{idx_suffix[i]}}_keys_limit1",
           "tags": ["lookup", "join", "limit1"],
           "clients": 1,
           "warmup-iterations": 10,
           "iterations": 50
         },
         {
-          "operation": "esql_lookup_join_1k_keys_limit1000",
+          "operation": "esql_lookup_join_{{idx_suffix[i]}}_keys_limit1000",
           "tags": ["lookup", "join", "limit1000"],
           "clients": 1,
           "warmup-iterations": 10,
           "iterations": 50
         },
         {
-          "operation": "esql_lookup_join_1k_keys_limit10000",
+          "operation": "esql_lookup_join_{{idx_suffix[i]}}_keys_limit10000",
           "tags": ["lookup", "join", "limit10000"],
           "clients": 1,
           "warmup-iterations": 10,
           "iterations": 50
         },
         {
-          "operation": "esql_lookup_join_1k_keys_keep_limit10000",
+          "operation": "esql_lookup_join_{{idx_suffix[i]}}_keys_keep_limit10000",
           "tags": ["lookup", "join", "limit10000"],
           "clients": 1,
           "warmup-iterations": 10,
           "iterations": 50
         },
         {
-          "operation": "esql_lookup_join_1k_keys_sort_limit10000",
+          "operation": "esql_lookup_join_{{idx_suffix[i]}}_keys_sort_limit10000",
           "tags": ["lookup", "join", "limit10000"],
           "clients": 1,
           "warmup-iterations": 5,
           "iterations": 20
         },
         {
-          "operation": "esql_lookup_join_1k_keys_where_limit1000",
+          "operation": "esql_lookup_join_{{idx_suffix[i]}}_keys_where_limit1000",
           "tags": ["lookup", "join", "limit1000"],
           "clients": 1,
           "warmup-iterations": 10,
           "iterations": 50
         },
         {
-          "operation": "esql_lookup_join_1k_keys_where_no_match",
+          "operation": "esql_lookup_join_{{idx_suffix[i]}}_keys_where_no_match",
```

Comment:
This enables the no match queries in this challenge to run against all indices, while currently this is commented out for all indices except the 1k one. I think this is not intended, as we know this task is particularly long running, right?

Reply:
Right. I'd like to be able to run them, but probably it's better to do it as a separate PR, and in a brand new "large" challenge.

Reply:
Done